DNN Platform/Tests/DotNetNuke.Tests.Web/Api/PortalAliasRouteManagerTests.cs

```diff
@@ -47,10 +47,10 @@ public void TearDown()
 
 
         [Test]
-        [TestCase("mfn", "url", 0, "DesktopModules/mfn/API/url")]
-        [TestCase("mfn", "url", 1, "{prefix0}/DesktopModules/mfn/API/url")]
-        [TestCase("mfn", "url", 2, "{prefix0}/{prefix1}/DesktopModules/mfn/API/url")]
-        [TestCase("fee/foo", "{contoller}/{action}/{id}", 4, "{prefix0}/{prefix1}/{prefix2}/{prefix3}/DesktopModules/fee/foo/API/{contoller}/{action}/{id}")]
+        [TestCase("mfn", "url", 0, "API/mfn/Action/url")]
+        [TestCase("mfn", "url", 1, "{prefix0}/API/mfn/Action/url")]
+        [TestCase("mfn", "url", 2, "{prefix0}/{prefix1}/API/mfn/Action/url")]
+        [TestCase("fee/foo", "{contoller}/{action}/{id}", 4, "{prefix0}/{prefix1}/{prefix2}/{prefix3}/API/fee/foo/Action/{contoller}/{action}/{id}")]
         public void GetRouteUrl(string moduleFolderName, string url, int count, string expected)
```

Comment:
this test is for GetRouteUrl, it always return new format, the backward compatibility implement in ServicesRoutingManager, add a new tes in ServicesRoutingManagerTest to check backward compability.